pyproject.toml

```diff
@@ -0,0 +1,62 @@
+[build-system]
+requires = ["hatchling", "pathspec"]
+build-backend = "hatchling.build"
+
+[project]
+name = "verificarlo"
+version = "0.9.1"
```

Comment:
Can we retrieve the version number from configure.ac automatically, it simplifies releases. Now we have to change the version number only in two places: configure.ac and README.md. I would prefer we keep it like that.

Reply:
Yes, you're right. I updated it, so now the version is in the `VERSION` file. It's read by `configure.ac` and `pyproject.toml`. We still have to update it manually in the `README.md` since there is no mechanism to read a file from it.